backend/src/main/java/com/bakdata/conquery/io/external/form/FormBackendVersion.java

```diff
@@ -0,0 +1,8 @@
+package com.bakdata.conquery.io.external.form;
+
+import lombok.Data;
+
+@Data
+public class FormBackendVersion {
+	private String version;
+}
```

Comment:
@awildturtok 
```suggestion
@Data
public class FormBackendVersion {
	private final String version;
}
```
Weißt du, weshalb er es so nicht deserializieren kann?

Reply:
Du musst den RequiredArgsConstructor mit `@JsonCreator` separat anhängen denke ich

Reply:
so macht jackson das über NoArgs+Setter?

Reply:
Komisch, denn Jackson sagt, dass er einen Creator findet, auch ohne die Creator-Annotation, aber will sie nicht nutzen. Aber gut.